pymc_marketing/mmm/lift_test.py

```diff
@@ -0,0 +1,366 @@
+        name: index_variable(
+            var_dims=model.named_vars_to_dims[var_name],
+            var=model[var_name],
+            indices=indices,
+        )
+        for name, var_name in variable_mapping.items()
+    }
+
+    partial_saturation_function = partial(saturation_function, **kwargs)
+    empirical_lift = empirical_lift_measurements(
+        x_before, x_after, partial_saturation_function
+    )
+
+    dist(
+        name=name,
+        mu=empirical_lift,
+        sigma=df_lift_test["sigma"].to_numpy(),
+        observed=df_lift_test["delta_y"].to_numpy(),
+    )
+
+
+def add_menten_empirical_lift_measurements_to_likelihood(
+    df_lift_test: pd.DataFrame,
+    alpha_name: str,
+    lam_name: str,
+    dist=pm.Gamma,
+    model: Optional[pm.Model] = None,
+    name: str = "lift_measurements",
+) -> None:
+    """Add empirical lift measurements to the likelihood of the model.
+
+    Args:
+        df_lift_test: DataFrame with lift test results.
+        alpha_name: Name of the alpha parameter in the model.
+        lam_name: Name of the lambda parameter in the model.
+        dist: PyMC distribution to use for the likelihood.
+        model: PyMC model with date and channel coordinates.
+        name: Name of the likelihood.
+
+    Returns:
+        None
+
+    """
+    variable_mapping = {
+        "alpha": alpha_name,
+        "lam": lam_name,
+    }
+
+    add_lift_measurements_to_likelihood(
+        df_lift_test,
+        variable_mapping,
+        saturation_function=michaelis_menten,
+        model=model,
+        dist=dist,
+        name=name,
+    )
+
+
+def add_logistic_empirical_lift_measurements_to_likelihood(
+    df_lift_test: pd.DataFrame,
+    lam_name: str,
+    beta_name: str,
+    dist=pm.Gamma,
+    model: Optional[pm.Model] = None,
+    name: str = "lift_measurements",
+) -> None:
+    """Add empirical lift measurements to the likelihood of the model.
+
+    Args:
+        df_lift_test: DataFrame with lift test results.
+        lam_name: Name of the lambda parameter in the model.
+        beta_name: Name of the beta parameter in the model.
+        dist: PyMC distribution to use for the likelihood.
+        model: PyMC model with date and channel coordinates.
+        name: Name of the likelihood.
+
+    Returns:
+        None
+
+    """
+    variable_mapping = {
+        "lam": lam_name,
+        "beta": beta_name,
+    }
+
+    def saturation_function(x, beta, lam):
+        return beta * logistic_saturation(x, lam)
+
+    add_lift_measurements_to_likelihood(
+        df_lift_test,
+        variable_mapping,
+        saturation_function=saturation_function,
+        model=model,
+        dist=dist,
+        name=name,
+    )
+
+
+def scale_channel_lift_measurements(
+    df_lift_test: pd.DataFrame,
+    channel_col: str,
+    channel_columns: list[str],
+    transform: Callable[[np.ndarray], np.ndarray],
+) -> pd.DataFrame:
+    """Scale the lift measurements for a specific channel.
+
+    Args:
```

Comment:
I think we are following the numpy docstring format right? https://numpydoc.readthedocs.io/en/latest/format.html